common/database.cpp

```diff
@@ -338,6 +338,27 @@ bool Database::ReserveName(uint32 account_id, char* name) {
 	query = StringFormat("INSERT INTO `character_data` SET `account_id` = %i, `name` = '%s'", account_id, name);
 	results = QueryDatabase(query);
 	if (!results.Success() || results.ErrorMessage() != ""){ return false; }
+	
+	// Put character into the default guild if rule is being used.
+	int guild_id = RuleI(Character, DefaultGuild);
+
+	if (guild_id != 0) {
+		int char_id=-1;
```

Comment:
Please use full `character_id` and use the ID from the insert above via `results.LastInsertedID()` instead of making an additional unnecessary call

Reply:
I had no idea this existed.   Changing it now.